src/pages/Download/sections/DownloadApp.tsx

```diff
@@ -3,6 +3,7 @@ import { GetInsuredButton } from 'components/get-insured-button'
 import { Field, Form, Formik } from 'formik'
 import * as React from 'react'
 import styled from 'react-emotion'
+import * as Yup from 'yup'
```

Comment:
imo `yup` should be lowercase but that's just a personal preference :) if we use upper Y i think we should change the other usages as well for consistency